test/inc/TestDef.h

```diff
@@ -23,6 +23,10 @@ static const std::wstring c_testRequestRestartNowRestartedPhasePackagedEventName
 
 static const std::wstring c_testPushPhaseEventName = L"WindowsAppRuntimeTestPushPhaseEventName";
 
+static const std::wstring c_testNotificationPhaseEventName = L"WindowsAppRuntimeTestNotificationPhaseEventName";
+static const std::wstring c_rawNotificationPayload = L"rawNotificationValue";
```

Comment:
Getting this error when building this branch:
13>BaseTestSuite.cpp
13>D:\WindowsAppSDK\test\inc\TestDef.h(27,27): error C2371: 'c_rawNotificationPayload': redefinition; different basic types
13>D:\WindowsAppSDK\test\PushNotificationTests\PushNotification-Test-Constants.h(5): message : see declaration of 'c_rawNotificationPayload'

Have all changes / files been pushed?

Reply:
fixed